src/components/Developer/UiLibrary/PivotCardsDemo.js

```diff
@@ -3,7 +3,8 @@ import {
   FiftyObservationCard,
   FirstObservationCard,
   NotificationOnboarding,
-  SecondObservationCard
+  SecondObservationCard,
+  TenObservationCard
```

Comment:
same naming feedback -- either `Ten` or `Tenth` is good as long as we're consistent with the other pivot cards.